installer/bundle/bundle_skel.sh

```diff
@@ -379,6 +381,24 @@ python_ctypes_installed() {
     return $hasCtypes
 }
 
+curl_installed() {
+    # Check for curl (required for omsconfig)
+    preReqCurl=1
+    which curl 1> /dev/null 2> /dev/null
+    [ $? -eq 0 ] && preReqCurl=0
+
+    return $preReqCurl
+}
+
+gpg_installed() {
```

Comment:
documented dsc requirements are out of scope for omsagent and unnecessary would require keeping these docs in sync. there is a separate repo for each of the dependencies.  So I would think that it should be doc'ed in corresponding repo.